examples/resources/hcp_azure_peering_connection/resource.tf

```diff
@@ -0,0 +1,73 @@
+resource "hcp_hvn" "hvn" {
+  hvn_id         = "main-hvn"
+  cloud_provider = "azure"
+  region         = "westus2"
+  cidr_block     = "172.25.16.0/20"
+}
+
+resource "hcp_azure_peering_connection" "peer" {
+  hvn_link                 = hcp_hvn.hvn.self_link
+  peering_id               = "dev"
+  peer_vnet_name           = azurerm_virtual_network.vnet.name
+  peer_subscription_id     = "50dc4fb8-db16-4c07-8c35-f31467122911"
+  peer_tenant_id           = "09cdb642-fb81-4425-ba72-b882790ecb06"
+  peer_resource_group_name = azurerm_resource_group.rg.name
+  peer_vnet_region         = azure_rm_virtual_network.region
+}
+
+resource "hcp_peering_connection_activation" "activation" {
+  peering_id = hcp_azure_peering_connection.peer.peering_id
+  hvn_link   = hcp_hvn.hvn.self_link
+}
+
+resource "hcp_hvn_route" "route" {
```

Comment:
Does this work with Azure? I am going to test now but curious if you noticed any problems with it.

Reply:
So far it has as long as the peering connection is already active.